tasks/main.yml

```diff
@@ -66,6 +68,11 @@
       check_mode: "no"
       register: host_info
 
+    - name: Set Volumes
+      set_fact:
+        volumes: "{{ ovirt_clusters[0].gluster_volumes }}"
+      when:  ovirt_clusters[0].gluster_service | bool
```

Comment:
when `ovirt_clusters[0].gluster_service` is disabled the `gluster_volumes` is empty list i think that we wont need the atribute `volumes`

Reply:
Okay.